### PR TITLE
added npm scripts

### PR DESCRIPTION
**Pull request by vutran**
This PR adds npm scripts to easily run from sources or build with a simple command, keeping the simplicity going forward if the build process becomes more complex.

### CODE REVIEW

**Comment by ewnd9** [comment]
I suggest to start the `watch` task https://github.com/appetizermonster/hain/blob/40ccb5a5ad16f67a03ed6221ac353c4883ca185a/gulpfile.js#L57 in a separate terminal for development instead of `gulp && electron .`

**Comment by appetizermonster** [comment]
@vutran Good, it's reasonable, but i think your Word-Case in docs is a little different with the original.
i think it's better to write `Build from Source` than `Build from source`, how do you think about it?

**Comment by vutran** [comment]
@appetizermonster I agree, sorry about that. I pushed a new commit with the pudaet.

File: README.md [hunk 1]
```diff
@@ -28,12 +28,13 @@ cinst Hain -pre
...
 ## Usage
 Run and press `alt+space` anywhere
-      
+
 ## How to make Plugins
 
 See [Plugin Documentation](docs/plugin-docs.md)
 
-## Running from Sources
+## Install/Build from source
```
Comment (appetizermonster) [inline README.md]: Please check this line as well :)

**Comment by appetizermonster** [comment]
@vutran Thanks! :+1:

**Pull request merged**